SECURITY.md

```diff
@@ -1,30 +1,37 @@
 # Security Policy
 
+<picture>
+    <!-- Avoid image being clickable with slight workaround --->
+    <source media="(prefers-color-scheme: dark)" srcset="docs/assets/lawnchair-round.webp" width="100">
+    <img alt="" src="docs/assets/lawnchair-round.webp" width="100">
+</picture>
```

Comment:
I don't think we need to have an icon for a page detailing security policies and measures.